### ISSUE TITLE
Data types are not getting explained in the doc

### ISSUE THREAD

**Issue by SetTrend** [less/less-docs#359]
The doc doesn't elaborate on the differenct LESS data types.

Samples of questions left unanswered after reading the doc:

  1. What's a list?
  1. What's a dimension?
  1. Are there other, LESS specific data types?
  1. How to retrieve a single value from a list?

**Comment by seven-phases-max** [comment]
1. Just comma or whitespace separated list of values.
2. Just a number with an optional unit (CSS [dimension](http://www.w3.org/TR/css3-values/#dimensions)).
3. CSS color is a Less specific type too.
4. [List Functions](http://lesscss.org/functions/#list-functions).

**Comment by SetTrend** [comment]
This should all be given in the documentation.

  1. Without documentation lists can be anything. Syntax should be specified.
  1. According to [Less image-size](lesscss.org/functions/#misc-functions-image-size), *dimension* are *two* values with a unit. Is this a typo?
  1. The allowed forms of color should also be specified in the documentation.
  1. There should be a *"see also"* hyperlink added everywhere in the documentation where list function elements are mentioned.

**Comment by seven-phases-max** [comment]
Well, in summary, the docs are open source too - so their consistency and completeness are totally dependent on people contributing to it. For instance notice a pencil icon at the beginning of most chapters/paragraphs - click it and you'll get to its corresponding source code right here at GitHub. Thus anyone with GitHub account) can edit and submit the changes w/o need of any special tools or editors (For more advanced editing see https://github.com/less/less-docs/blob/master/CONTRIBUTING.md).

In general, note the major Less contributors are as plain Less user as you.
---
1. In general it's assumed a reader is familiar with CSS (where "a list" is pretty unambiguous thing)
2. That's just a slight mix up  (someone just merged two different meaning of the same word in the same docs, i.e. Less dimension (which is in fact only internal name and probably should not be used in the documentation at all (a "number" to be used instead)) with plain English dimension (as in "image dimensions". Hense the confusion).
3. Just all standard CSS color values.
4. See summary above.

**Comment by SetTrend** [comment]
It's not clear from the docs, however, whether these are CSS types or LESS types - which might be different.

I'm a LESS consumer, not a LESS contributor. And I strongly believe that going public includes the programmer's responsibility to provide proper documentation.

**Comment by seven-phases-max** [comment]
>And I strongly believe that going public includes the programmer's responsibility to provide proper documentation.

In a perfect world only. In practice Less contributors (that are as same plain Less consumers yet again) don't have infinite amount of time to cover everything in the docs, so normally it's usually assumed OK for you to waste several minutes to ask about something unclear (at the [SO](http://stackoverflow.com/questions/tagged/less) for example), in exchange to many days and weeks already put by those guys into the language and the docs.

In other words, I'm not arguing your "should be's", I'm just hinting that it's just impossible to cover everything (so that it's fine to create "Document List Type" ticket here, but it's not going to appear in the docs until some volunteer actually sits and writes that. Alternative is to become draconish and require every contributor to actually document everything they write in details, so that we have to instantly remove the List features and (and many many others) from the language because "it's not well documented").

**Comment by SomMeri** [comment]
@SetTrend I see your point, I think that documentation is important too.

We would really welcome you as a contributor. Not everyone can write well enough and fast enough to make worthwhile - people willing and able to write well are welcome on most projects I know about. Many people who would be willing to spend time do not even know where to start and where the fault lies nor how to fix it. So, to concluded, help with writing would be appreciated.
